LEGO1/lego/legoomni/include/legoworldlist.h

```diff
@@ -65,7 +65,7 @@ class LegoWorldListCursor : public MxPtrListCursor<LegoWorld> {
 // TEMPLATE: LEGO1 0x10059900
 // MxCollection<LegoWorld *>::~MxCollection<LegoWorld *>
 
-// TEMPLATE: LEGO1 0x10059950
+// TEMPLATE: LEGO1 0x10059947
```

Comment:
I would like to have a second opinion on this. I am a little puzzled by this address that is not divisible by 16, but the code looks like this should be the correct location.

Reply:
`0x10059950` is correct. It's a function with only a `ret` instruction, and it is referenced in the `MxCollection` constructor for that template type.

`0x10059947` is an exception handler unwind from the function above (starting at `0x10059900`). IIRC the shared ghidra had all the unwinds marked. Maybe someone had a script to do that? They are easy to find by reading all the [funcinfo](https://www.openrce.org/articles/full_view/21) structs